content/doc/developer/publishing/wiki-page.adoc

```diff
@@ -36,6 +36,11 @@ There are the following migration steps:
 . Modify the URL documentation page reference in the project file so that it points to GitHub (link:/doc/developer/publishing/documentation/#referencing-the-documentation-page-from-the-project-file[documentation]).
 . Commit changes, push them to your fork and create a pull request against the repository.
 
+NOTE: After migration, the continued existence of out-of-date wiki pages causes a number of problems, such as duplication, confusion, and unclear migration progress. 
+This persistence not only makes the process more difficult overall, but it also makes it harder to identify which pages have already been migrated and which are still waiting. 
+This is a major technical obstacle in the way of content migration from the wiki. 
+When migrating documentation, after the material has been properly moved, the out-of-date wiki page must be removed and a redirect must be configured to the migrated version. 
+
```

Comment:
```suggestion
NOTE: After migration, the continued existence of out-of-date wiki pages causes several problems, such as duplication, confusion, and unclear migration progress.
This persistence makes it harder to identify which pages have migrated and those waiting for migration.
If you cannot identify migrated pages, this creates a major technical obstacle in content migration from the wiki.
When migrating documentation, the out-of-date wiki page must be removed and have a redirect configured to point to the migrated version.

```
I have provided a suggestion to rework the content so that it reads a bit more smoothly, reduces unnecessary words/phrases, and uses active voice. I've also removed any trailing spaces left at the end of lines.